draft-zip.md

```diff
@@ -0,0 +1,97 @@
+
+This ZIP takes a much more explicit and non-optional approach, mandating at the protocol level that 50% of transaction fees be deposited into the ZSF As noted above, implementing this change allows the ZSF to accrue value earlier and contribute to future network sustainability.
+
+This system currently looks something like this:
+
+At Every New Block:
+- `ZSF_DEPOSIT` amount is deposited into the `ZSF_BALANCE_AFTER[h]`
+- Miner rewards come from `ZSF_BALANCE_AFTER[h]`
+- Transaction fees (inputs - outputs) paid to miner
+
+After the features described in this ZIP are activated (changed parts in bold):
+
+At Every New Block:
+- `ZSF_DEPOSIT` amount is deposited into the `ZSF_BALANCE_AFTER[h]`
+- Miner rewarded from `ZSF_BALANCE_AFTER[h]`
+- 50% of transaction fees (inputs - outputs) paid to miner
+- 50% of transaction fees deposited into `ZSF_BALANCE_AFTER[h]`
+
+This has a multitude of benefits:
+
+1. **Network Sustainability**: This mechanism involves temporarily reducing the supply of ZEC similar to asset burning in Ethereum’s EIP-1559, but with potential long-term sustainability benefits as the redistribution of deposits contributes to issuance rewards and network development, making it an attractive option for current and future Zcash users.
+2. **Ecosystem Benefits of Longer Time Horizons**: A reliable and long-term functioning Zcash blockchain allows users to make secure long-term plans, leading to a sustainable and virtuous adoption cycle, rather than being influenced by short-term trends.
+3. **Incentivizing Transaction Inclusion**: By providing a 10% share of transaction fees to miners, this ZIP maintains incentives for miners to prioritize including transactions in their blocks. This helps ensure the efficient processing of transactions and supports a robust and responsive network.
+4. **Future-Proofing the Network**: Diverting transaction fees into the `ZSF_BALANCE_AFTER[h]` is a forward-looking approach that prepares the Zcash network for future challenges and opportunities. It establishes a financial buffer that can be instrumental in addressing unforeseen issues and seizing strategic advantages as the Zcash ecosystem evolves.
+
+# Specification
+
+This ZIP only proposes a single modification to the transaction fees:
+1. Keep the current destination of 10% of the fees untouched, but route 50% of the fees back to `ZSF_BALANCE_AFTER[h]`
+
+Please note that a network upgrade is required for this work to be fully implemented.
+
+## Transaction fee routing requirements
+
+- For each transaction, 50% of the total fee MUST be paid to the `ZSF_BALANCE_AFTER[h]`
+- The minimum of the 50% fee MUST equal 1 zatoshi or more.
+
+# Rationale
+
+We believe that is ultimately a very minor change to the protocol, and quite simple in terms of implementation overhead. Additionally – and at the time of this writing – transaction fees are so small that 50% will likely not have a major impact.
```

Comment:
Like PR #706, it might be helpful to have a graph that shows when the difference would be significant at the current fee rate.

Reply:
See my reply below. I've added a [section](https://github.com/eigerco/zips/blob/ZSF-50-50/draft-zip.md#estimated-impact-on-miners) that goes through the numbers. I'm not sure a graph would add much here so I've skipped it for now.